### DISCUSSION TITLE
Sellbrite-API: /Inventory (GET) Only Loads 100 lines

### DISCUSSION THREAD

**Discussion by nellayroc** [Sellbrite/Sellbrite-API#41] [Q&A]
Hello,

Simple question. Using the /inventory (GET) endpoint to bring all of my data into a Google sheet. However, it only seems to load 100 rows/entries. Is there a way I can bring in all lines of inventory (it should only be a few hundred rows of data)?

Thanks

**Comment by electrictomcat** [comment]
When you access the inventory get endpoint you can pass a page parameter which will bring the next 100 products.
There will also be a total-pages header which tells you how many pages there are.

**Reply by nellayroc** [reply]
Thanks for the help, here. A bit new to API - do you happen to know what the syntax would look like?

**Reply by electrictomcat** [reply]
It depends on what you are using to make the call.

In Postman there is a section for Parameters and you would simply add
Page => <some #>

In most languages you add paramenters in array format so it would be like:
[ 'page' => 1 ]
